services/nvidia-gpu-operator/1.11.1/defaults/cm.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nvidia-gpu-operator-1.11.1-d2iq-defaults
+  namespace: ${releaseNamespace}
+data:
+  values.yaml: |-
+    nfd:
+      enabled: false
+    driver:
+      enabled: false
+    toolkit:
+      version: v1.10.0-centos7
+    gfd:
+      enabled: false
```

Comment:
```suggestion
```

Reply:
we can tweak the values in the configuration task. this is just a value we already have some insights on.

as it turns out, v1.11.1 of the gpu operator app doesn't even support turning it off. 

i also thought we agreed to have gpu operator deploy its own gfd from this thread https://mesosphere.slack.com/archives/C01MPKVBB5E/p1663188128968139

Reply:
Ah I see, I didn't realize this was the field we wanted to leave on or that it's not even supported in this release (I just carried over the values from the doc). And if I'm being completely honest, I also didn't want to make all these PRs run through CI again 😆  but given that `enabled: false` isn't even disabling anything now, I'll fix it to at least read `true` so when we upgrade to a version that supports it, there aren't any surprises there since we do want it to be turned on. I'll update + rebase